### ISSUE TITLE
hadolint: :Dockerfile:: openBinaryFile: does not exist (No such file or directory)

### ISSUE THREAD

**Issue by droslean** [hadolint/hadolint#877]
- [x] This is a bug report
- [x] I searched existing issues before opening this one

### Expected behavior

I am expecting handling to find all Dockerfiles in a folder and lint them.

### Actual behavior
It trying to find a specific Dockerfile in the root folder

### Steps to reproduce the behavior

hadolint Dockerfile* --verbose

I tried also `^Dockerfile$` and `*Dockerfile*`

Haskell Dockerfile Linter 2.10.0

**Comment by m-ildefons** [comment]
Hi Nikolaos,

Hadolint relies on the user to tell it which files to lint. If you don't want to give an explicit list of files, but rather want something to automatically discover them based on a naming pattern, you can use the usual shell tooling to do so, e.g.:
```bash
$ find . -iname "*dockerfile*" -exec hadolint {} \;
```
The debian and alpine based container images come with a variety of tools to do so.
I hope that works for you.

**Comment by droslean** [comment]
@m-ildefons Yea I am already using this approach, unfortunately.  I opened the issue to give some heads up about the behavior and wait for confirmation if the feature will be implemented. I guess not, so leaving this issue opened is just a waste of time. Thanks.
